hypervisor/src/kvm/x86_64/mod.rs

```diff
@@ -53,6 +53,9 @@ pub fn check_required_kvm_extensions(kvm: &Kvm) -> KvmResult<()> {
     if !kvm.check_extension(Cap::GetTscKhz) {
         return Err(KvmError::CapabilityMissing(Cap::GetTscKhz));
     }
+    if !kvm.check_extension(Cap::ReinjectControl) {
+        return Err(KvmError::CapabilityMissing(Cap::ReinjectControl))
+    }
```

Comment:
It appears that AVIC requires PIT re-injection to be disabled. I haven't found where we actually create the PIT, so this might be neccessary.